src/service-worker.11ty.js

```diff
@@ -0,0 +1,42 @@
+const glob = require("glob")
+
+/**
+ * Generates the service worker that browsers use for app-level caching.
+ * Only caches images because they are responsible for the majority of the bandwidth usage but rarely change.
+ * When an existing image changes, update the `version` below, or else past visitors will only see the old image.
+ */
+class ServiceWorker {
+    data() {
+        return {
+            version: 1, // <-- !!! Increment the number if any _existing_ image files changed !!!
```

Comment:
Maybe I would put version: 1 into variable on top of the file like
```javascript
const VERSION_PREFIX = 1
```
and then I would add pre-build script to change that number in the file with sed, awk or something like that. 

This makes it so we make new cache everytime there is new build, and therefore we can cache js files as well, beside service-worker.11ty.js

Reply:
I get you point,  I only suggested this because the project is not as active, therefore does not received many updates which makes your concern not as impactfull. It is also less work and thing to pay attention to. 

I am neutral in this. Both solusions work.

Reply:
Thinking about it some more, if we want the cache to be invalidated on every build, then perhaps we could just make 11ty generate a random hash every time.

Reply:
It does not matter how we do it, I just wanted to add js files as well. Some of them are rather large. 

anyway, it is crashing in heroku environment. You might need to spin up your own heroku to see what is up. I did that before to figure out.